akka-stream-tests-tck/src/test/scala/akka/stream/tck/Timeouts.scala

```diff
@@ -12,8 +12,8 @@ import akka.testkit._
  */
 object Timeouts {
 
-  def publisherShutdownTimeoutMillis: Int = 1000
+  def publisherShutdownTimeoutMillis: Int = 3000
 
-  def defaultTimeoutMillis: Int = 500
+  def defaultTimeoutMillis: Int = 800
```

Comment:
Dependency tracking builds would help, also known as the Holy Grail :<

Let's increase for now, but in medium term I think we must invest time on paralleising the build into "stages" and those must be parallel on multiple machines.